DigitalLearningSolutions.Web/Views/LearningPortal/Current/_CurrentLearningResourceCard.cshtml

```diff
@@ -31,7 +31,9 @@
         </div>
       </div>
       <a class="nhsuk-button nhsuk-button--secondary small-edit-button nhsuk-u-margin-bottom-4"
-         aria-describedby="@Model.Id-name-lhr">
+         aria-describedby="@Model.Id-name-lhr"
+         asp-action="MarkActionPlanResourceAsComplete"
+         asp-route-learningLogItemId="@Model.Id">

```

Comment:
There's a minor merge conflict here about the role="button" that I added in the 659 fix.

Reply:
fixed!